calm_adapter/calm_api_client/src/test/scala/uk/ac/wellcome/platform/calm_api_client/fixtures/CalmResponseGenerators.scala

```diff
@@ -0,0 +1,62 @@
+package uk.ac.wellcome.platform.calm_api_client.fixtures
+
+import java.time.Instant
+
+import akka.http.scaladsl.model.{DateTime, HttpProtocols, HttpResponse}
+import akka.http.scaladsl.model.headers.{`Set-Cookie`, Date, HttpCookie}
+
+import scala.xml.XML
+
+trait CalmResponseGenerators {
+
+  val cookie = ("cookie-name", "cookie-value")
+  val protocol = HttpProtocols.`HTTP/1.0`
+  val retrievedAt = Instant.ofEpochSecond(123456)
+
+  def searchResponse(
+    n: Int,
+    cookiePair: Option[(String, String)] = Some(cookie)): HttpResponse =
+    HttpResponse(
+      200,
+      cookiePair.map {
+        case (name, value) => `Set-Cookie`(HttpCookie(name, value))
+      }.toList,
+      <soap:Envelope
+      xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+        <soap:Body>
+          <SearchResponse xmlns="http://ds.co.uk/cs/webservices/">
+            <SearchResult>{n}</SearchResult>
+          </SearchResponse>
+        </soap:Body>
+      </soap:Envelope>.toString,
+      protocol
+    )
+
+  def summaryResponse(
+    data: List[(String, String)],
+    timestamp: Option[Instant] = Some(retrievedAt)): HttpResponse =
```

Comment:
I can't find any instances of this argument being set to a non-default value, maybe ditch it?

You could then do something like:

```scala
def summaryResponse(data: (String, String)*)
```

which IMO makes it slightly nicer when it's called.